README.md

```diff
@@ -28,25 +23,33 @@ It assumes a running SonarQube server or a project configured on SonarCloud.
 In order for the analysis to run, analysis properties need to be defined. 
 There are multiple ways of providing these properties, described below in descending order of priority:
 
-* Through CLI arguments to the `pysonar` command
-* Under the `[tool.sonar]` key of the `pyproject.toml` file
-* Through common properties extracted from the `pyproject.toml`
-* In a dedicated `sonar-project.properties` file
-* Through environment variables
+1. Through CLI arguments to the `pysonar` command
+2. Environment variables for individual properties (e.g. `SONAR_TOKEN`, `SONAR_VERBOSE`, `SONAR_HOST_URL`, ...)
+3. Generic environment variable `SONAR_SCANNER_JSON_PARAMS` 
+4. Under the `[tool.sonar]` key of the `pyproject.toml` file
+5. In a dedicated `sonar-project.properties` file
+6. Through common properties extracted from the `pyproject.toml`
 
 ### Through CLI arguments
 
 Analysis properties can be provided as CLI arguments to the `pysonar` command.
-They follow the same convention as when running the SonarScanner CLI directly 
-(see [documentation](https://docs.sonarsource.com/sonarqube/9.9/analyzing-source-code/scanners/sonarscanner/#running-from-zip-file)).
+They can be provided in a similar way as when running the SonarScanner CLI directly 
+(see [documentation](https://docs.sonarsource.com/sonarqube-server/2025.1/analyzing-source-code/scanners/sonarscanner/#running-from-zip-file)).
 This means that analysis properties provided that way should be prepended with `-D`, for instance:
 
 ```
-$ pysonar -Dsonar.login=myAuthenticationToken 
+$ pysonar -Dsonar.token=myAuthenticationToken 
 ```
 
-You can use all the argument allowed by __SonarScanner__. 
-For more information on __SonarScanner__ please refer to the [SonarScanner documentation](https://docs.sonarsource.com/sonarqube/9.9/analyzing-source-code/scanners/sonarscanner/)
+You can use all the arguments allowed by __SonarScanner__. 
```

Comment:
It could be worth it to mention that we support also Python specific properties like `sonar.python.version`

Reply:
I'm tempted to think it's implied, since `CLI_ARGS.md` mentions them explicitly. 
We could keep a list of Python specific properties documented separately though, but I feel it might be overkill at this point?